babyAndkids.html

```diff
@@ -0,0 +1,379 @@
+                                <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+                                    <li><a class="dropdown-item" href="./mobile.html">Mobile</a></li>
+                                    <li><a class="dropdown-item" href="./books.html">Books</a></li>
+                                    <li><a class="dropdown-item" href="./">TV/Appliances</a></li>
+                                    <li><a class="dropdown-item" href="#">Furniture</a></li>
+                                    <li><a class="dropdown-item" href="./babyAndkids.html">Baby/Kids</a></li>
+                                    <li><a class="dropdown-item" href="./stationery.html">Stationery</a></li>
+                                    <li>
+                                        <hr class="dropdown-divider">
+                                    </li>
+
+                                    <li><a class="dropdown-item" href="./login.html">Login/Signup</a></li>
+                                </ul>
+                            </li>
+                            <li class="nav-item text-capitalize">
+                                <a class="nav-link" href="cart.html">My Cart</a>
+                            </li>
+                        </ul>
+                        <form class="d-flex">
+                            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+                            <button class="btn btn-outline-dark" type="submit">Search</button>
+                        </form>
+                </div>
+            </div>
+        </nav>
+    </section>
+
+
+        <header>
+          <div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel">
+            <div class="carousel-inner">
+              <div class="carousel-item active">
+                <img src="./Images/babyAndkids/2--.jpg" class="d-block w-100" alt="image4">
+              </div>
+              <div class="carousel-item">
+                <img src="./Images/babyAndkids/4.jpg" class="d-block w-100" alt="image5">
+              </div>
+              <div class="carousel-item">
+                <img src="./Images/babyAndkids/2.jpg" class="d-block w-100" alt="image6">
+              </div>
+            </div>
+            <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="prev">
+              <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+              <span class="visually-hidden">Previous</span>
+            </button>
+            <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="next">
+              <span class="carousel-control-next-icon" aria-hidden="true"></span>
+              <span class="visually-hidden">Next</span>
+            </button>
+          </div>
+          <br>
+        </header>
+   
+
+    <section class="offer">
```

Comment:
first add a class babyandkids
update this to 

`<section class="offer babyandkids">`